### ISSUE TITLE
Rearrange client cli init so methods appear in alphabetical order in help

### ISSUE THREAD

**Issue by dcolligan** [ga4gh/server#989]
Appears like this right now:
```
    help                show this help message and exit
    variants-search     Search for variants
    variantsets-search  Search for variantSets
    variantannotations-search
                        Search for variantAnnotaions.
    variantannotationsets-search
                        Search for variantAnnotationSets.
    variantsets-get     Get a variantSet
    referencesets-search
                        Search for referenceSets
    references-search   Search for references
    readgroupsets-search
                        Search for readGroupSets
    callsets-search     Search for callSets
    reads-search        Search for reads
    datasets-search     Search for datasets
    referencesets-get   Get a referenceset
    references-get      Get a reference
    readgroupsets-get   Get a read group set
    readgroups-get      Get a read group
    callsets-get        Get a callSet
    variants-get        Get a variant
    datasets-get        Get a dataset
    references-list-bases
                        List bases of a reference
```

**Comment by macieksmuga** [comment]
Can you wait with this until I add sequence annotations back in (coming very soon)? I had to add quite a bit of code to `cli.py` and `client.py` for it, and don't want an overly-messy merge, as may happen if significant parts of those files are rearranged.

**Comment by dcolligan** [comment]
It should be an easy merge; the changes are mostly line additions in the common section: #991

**Comment by dcolligan** [comment]
Fixed in #991
